### PR TITLE
Staging

### PR DESCRIPTION
**Pull request by Seyi-Keye**

### CODE REVIEW

File: .eslintignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+node_modules/*
+karma.conf.js
```
Comment (andela-aolaniran) [inline .eslintignore:2]: You might want eslint to ignore your .yml files and .gitignore file

**Review by andela-aolaniran — COMMENTED** [review]
Take note and fix all code style issues flagged by hound.

File: .gitignore [hunk 2]
```diff
@@ -1 +1,3 @@
-node_modules
\ No newline at end of file
+node_modules
+coverage
+.coveralls.yml
```
Comment (andela-uenabulele) [inline .gitignore]: **Nitpick:**
Configure your editor to add a linebreak at the end of your file on save

File: README.md [hunk 3]
```diff
@@ -1 +1,28 @@
 # inverted-index
+[![Coverage Status](https://coveralls.io/repos/github/andela-oaromokeye/checkpoint-inverted-index/badge.svg?branch=staging)](https://coveralls.io/github/andela-oaromokeye/checkpoint-inverted-index?branch=staging)
+[![Code Climate](https://codeclimate.com/github/andela-oaromokeye/checkpoint-inverted-index/badges/gpa.svg)](https://codeclimate.com/github/andela-oaromokeye/checkpoint-inverted-index)
+[![Build Status](https://travis-ci.org/andela-oaromokeye/checkpoint-inverted-index.svg?branch=master)](https://travis-ci.org/andela-oaromokeye/checkpoint-inverted-index)
+
+## Introduction
+Inverted index takes a JSON array of text objects and creates an index from the array. The index allows a user to search for text blocks in the array that contain a specified collection of words.
+
+## How can I get started with the app
+
```
Comment (andela-uenabulele) [inline README.md]: **Critical**
How can I get started with the app?

File: frontend/inverted-index.js [hunk 4]
```diff
@@ -0,0 +1,126 @@
+/**
```
Comment (andela-uenabulele) [inline frontend/inverted-index.js]: adding `/*eslint-disable no-unused-vars*/` will fix that houndCI issue
http://eslint.org/docs/user-guide/configuring

File: frontend/inverted-index.js [hunk 5]
```diff
@@ -0,0 +1,126 @@
...
+ */
+class InvertedIndex {
+  /**
+   * class constructor
+   * @constructor
+   */
+  constructor() {
+    this.allFiles = {};
+    this.index = {};
+    this.docNumber = {};
```
Comment (andela-uenabulele) [inline frontend/inverted-index.js]: **Nitpick**
do not use abbreviations for variable naming
https://github.com/andela/code-review-guidelines/tree/master/coding-style-guides

File: jasmine/spec/inverted-index-test.js [hunk 6]
```diff
@@ -0,0 +1,50 @@
+const book = [
```
Comment (andela-uenabulele) [inline jasmine/spec/inverted-index-test.js]: **Nitpick**
You might want to place your dummy data in a different file

File: package.json [hunk 7]
```diff
@@ -0,0 +1,55 @@
...
+    "eslint-config-airbnb-base": "^11.0.1",
+    "jshint": "^2.9.4"
+  },
+  "dependencies": {
+    "browser-sync": "^2.18.6",
+    "browserify": "^14.0.0",
+    "browserify-shim": "^3.8.13",
+    "connect": "^3.5.0",
+    "coveralls": "^2.11.15",
+    "eslint": "^3.14.1",
```
Comment (andela-uenabulele) [inline package.json:18]: **Nitpick**
eslint appears in both dependencies and devdependencies
I think these should be in dev dependencies too
+ "eslint-config-airbnb": "^14.0.0",
 +    "eslint-plugin-import": "^2.2.0",
 +    "eslint-plugin-jsx-a11y": "^3.0.2",
 +    "eslint-plugin-react": "^6.9.0",`
there might be others.

http://zhiye.li/2014-06-23-npm-dependencies-and-devDependencies.html
http://stackoverflow.com/questions/18875674/whats-the-difference-between-dependencies-devdependencies-and-peerdependencies

**Pull request closed without merging**